src/video/cocoa/cocoa_v.mm

```diff
@@ -697,6 +701,8 @@ - (void)updateLayer
 {
 	if (driver->cgcontext == nullptr) return;
 
+	PerformanceMeasurer framerate(PFE_VIDEO);
```

Comment:
Why this entry here? Feels out of place? The other drivers only have one place, and that is in `Paint`?

Reply:
It is part of drawing and runs on the main thread.

Reply:
Do you know if the performance measurer understands this? Or that it will average both calls now and fake that painting takes shorter than it really does?

Reply:
Good question. The number got higher for me, but I don't really see any summation in the code.

Reply:
The intention of PFE_VIDEO is mainly to measure delay between calls (that's the screen refresh rate) and less to measure how long the transfer of the graphics data from our buffer to the windowing system buffer takes. So as long as each use of PFE_VIDEO represents a screen refresh I'd say it's correct.

Reply:
Then it is not correct in the PR.